action.yml

```
@@ -13,6 +13,10 @@ inputs:
     description: "Sets the directory to run the benchmarks, relative to the project directory"
     required: false
     default: ""
+  package:
+    description: "Sets an specific package to run the benchmarks"
```

```suggestion
    description: "Benchmark only the specified packages"
```
Taken from [The Cargo book](https://doc.rust-lang.org/cargo/commands/cargo-bench.html)